agent/lib/kontena/websocket_client.rb

```diff
@@ -171,6 +172,7 @@ def on_close(event)
         handle_invalid_version
       end
       info "connection closed with code #{event.code}"
+      notify_actors('websocket:close', nil)
```

Comment:
The agent was not notifying anything if the server closed the connection, as opposed to the `websocket:disconnect` notification when the agent keepalive ping times out and the agent decides to close the connection.